website_upload_video/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Upload video on website
+=======================
+
+This module adds a "Upload video" option to the "Select a Media / Video" dialog.
+
+Usage
+=====
+
+On website:
+
+* click "Edit" button;
+* click add image button;
+* click "Video" tab;
+* click "Upload a video from your computer";
+* click "Save" button.
+
+Known issues / Roadmap
+======================
+
+* No video preview
+* Option autoplay is not implemented
+ 
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/website/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+website/issues/new?body=module:%20
+website_calendar_snippet%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```

Comment:
You are not putting any icon, so remove this part. Or better, create an icon with a film roll and an up arrow to represent the module :wink: